riscv-asm.md

```diff
@@ -183,6 +183,116 @@ Directive    | Arguments                      | Description
 .balign      | b,[pad_val=0]                  | byte align
 .zero        | integer                        | zero bytes
 
+### <a name=.option></a> `.option`
+
+#### `rvc`/`norvc`
+
+Enable/disable C-extension for following code region, this option is equivalent
+to `.option arch +c`/`.option arch -c`, but widely support by older version of
+toolchain.
+
+#### `arch`
+
+Enable and/or disable specific ISA extensions for following code regions, but
+without changing the arch attribute, that means it won't raise the minimal
+execution environment requirement, so the user should take care to protect
+execution of the code regions around `.option push`/`.option arch`/`.option pop`.
+A typical use case is is with `ifunc`, e.g. the libc is built with `rv64gc`,
+but a few functions like memcpy provide two versions, one built with `rv64gc`
+and one built with `rv64gcv`, and then select between them by ifunc mechanism
+at run-time.  However we don't want to change the minimal execution environment
+requirement to `rv64gcv`, since the `rv64gcv` version will be invoked only if
+the execution environment supports the vector extension, so the minimal
+execution environment requirement still is `rv64gc`.
```

Comment:
This should be commentary.

Reply:
If this is changed to commentary, it's important that the part about "won't raise the minimal execution environment requirement" be left as normative. We don't want to set `EF_RISCV_RVC` in the ELF flags when using `.option arch, +c`, even though binutils currently does that for `.option rvc`. See https://reviews.llvm.org/D122490#3419774 for a recent discussion about that.